jsx/InfoPanel.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Display a message in an information panel.
+ *
+ * @param {object} props - React props
+ *
+ * @return {JSX}
+ */
+function InfoPanel(props) {
+    return (
+        <div className="alert alert-info"
+           style={{
+             backgroundColor: '#BEDFFF',
+             color: 'black',
+             display: 'flex',
+             flexBasis: 'row',
+             flexWrap: 'nowrap',
+         }}>
+            <div style={{
```

Comment:
Hi @driusan it's better to use <span> tag instead of <div> tag here.

Reply:
Why would `span` be better than `div`? They're block elements, not inline.

Reply:
i'm not sure if this is an opinion that i have - i think div tags are fine to use!